archai/nlp/models/model_mixed_qat.py

```diff
@@ -19,7 +19,7 @@ class MixedQATModel(ArchaiModel):
 
     """
 
-    def __init__(self, model: ArchaiModel, qat_weight: Optional[float] = 0.2) -> None:
+    def __init__(self, model: ArchaiModel, qat_weight: Optional[float] = 0.2, diffp: Optional[bool] = False) -> None:
```

Comment:
Why is this Optional?

Reply:
It is not, I was using Optional incorrectly. Will fix.